homeassistant/components/webostv/quality_scale.yaml

```diff
@@ -0,0 +1,61 @@
+rules:
```

Comment:
Possible clean up point: `self._attr_device_info` is only read once: when the entity is added. Currently the logic to fetch the device specific data is in `_update_states`, so in theory it doesn't do much after the entity has been added